ports/libdshowcapture/portfile.cmake

```
@@ -1,11 +1,25 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO obsproject/libdshowcapture
-    REF cba07c63810f51a58f6fb7f2e3b0fb162b5a6313
-    SHA512 962f5886f637f06580db9b90d238cdb76976846c5b1d49112910fda0da689788abec1d1703aa4e91ee4be57f328eb8183c04f94119662e1243269ae66f023c84
+    REF 8878638324393815512f802640b0d5ce940161f1
+    SHA512 bbb9fa169bffce4f6405b8332524267f10b3e6e2dcaddcddf7ef73ffb7a6409ef4c6a13f599cab814cbf42c22690f9e24e988666886535ef9fdfb851fdb50a5c
     HEAD_REF master
+    PATCHES
+        fix_build.patch
 )
 
+vcpkg_from_github(
+    OUT_SOURCE_PATH DEP_SOURCE_PATH
+    REPO elgatosf/capture-device-support
+    REF fe9630974d47f51bf54826e72fb8b654e620aa93
+    SHA512 971185ffaf0c5777c060d3cf49ee8f907aebc8191e3ada9c9f3c4c0d553c257d13e2828c991985b9d47a446d003b26664ecec2c18c0e6c66dfdba904baee0ae6
+    HEAD_REF master
```

```suggestion
    HEAD_REF main
```
The value of HEAD_REF is usually set to the default branch name of the repository